README.md

```diff
@@ -23,15 +23,15 @@ The **first way** to use Cost-of-Capital-Calculator is to download the source co
 * You can adjust the `example.py` by adjusting the individual income tax reform (using a dictionary or JSON file in a format that is consistent with [Tax Calculator](https://github.com/PSLmodels/Tax-Calculator)) or other model parameters specified in the `business_tax_adjustments` dictionary.
 * Model outputs will be saved in the following files:
   * `./baseline_byasset.csv`
-    * Cost of capital, marginal effective tax rates, effective average tax rates, and other model output for the baseline policy, organized by asset.
+    * Cost of capital, marginal effective tax rates, effective average tax rates, and other model output for the baseline policy, organized by an asset.
```

Comment:
The output is organized by type of asset.  So perhaps this could read, "...organized by type of asset."